sdk-extensions/jaeger-remote-sampler/src/test/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerIntegrationTest.java

```diff
@@ -38,13 +37,12 @@
 
   @Test
   void remoteSampling_perOperation() {
-    String jaegerHost =
-        String.format("127.0.0.1:%d", jaegerContainer.getMappedPort(COLLECTOR_PORT));
     final JaegerRemoteSampler remoteSampler =
-        JaegerRemoteSampler.builder()
-            .setChannel(ManagedChannelBuilder.forTarget(jaegerHost).usePlaintext().build())
-            .setServiceName(SERVICE_NAME)
-            .build();
+        (JaegerRemoteSampler)
```

Comment:
any reason to not have the builder return the concrete class, rather than the interface? Is that for the agent code to work more easily?

Reply:
I thought it could make things more future-proof, but after a rethink, I can't come up with a benefit indeed, reverted.